docs/python_docs/python/scripts/conf.py

```diff
@@ -237,6 +234,16 @@
 # let the source file format to be xxx.ipynb instead of xxx.ipynb.txt
 html_sourcelink_suffix = ''
 
+html_context = {
+    'display_github': True,
+    'github_user': 'apache',
+    'github_repo': 'mxnet',
```

Comment:
incubator-mxnet?

Reply:
There is forward in the link and I think the link is future proof 😆